spec.md

```diff
@@ -218,6 +218,48 @@ in context attributes, including in extension attributes.
 
 The following attributes are REQUIRED to be present in all CloudEvents:
 
+#### datacontenttype
```

Comment:
What if instead of putting it under the "required" section, we keep it down below but rename that section to something like "Conditional/Optional Attributes" ?